lib/faexport/scraper.rb

```diff
@@ -489,7 +489,234 @@ def submit_journal(title, description)
+  def notifications(include_deleted)
+    # Get page code
+    html = fetch("msg/others/")
+    # Parse page
+    login_user = get_current_user(html)
+    # Parse new watcher notifications
+    new_watches = []
+    watches_elem = html.at_css("ul#watches")
+    if watches_elem
+      watches_elem.css("li:not(.section-controls)").each do |elem|
+        if elem.at_css("input")['checked'] == "checked"
+          if include_deleted
+            new_watches << {
+                watch_id: "",
+                name: "Removed by the user",
+                profile: "",
+                profile_name: "",
+                avatar: fa_url(elem.at_css("img")['src']),
+                posted: "",
+                posted_at: ""
+            }
+          end
+          next
+        end
+        date = pick_date(elem.at_css('.popup_date'))
+        new_watches << {
+            watch_id: elem.at_css("input")['value'],
+            name: elem.at_css("span").content,
+            profile: fa_url(elem.at_css("a")['href']),
+            profile_name: last_path(elem.at_css("a")['href']),
+            avatar: "https:#{elem.at_css("img")['src']}",
+            posted: date,
+            posted_at: to_iso8601(date)
+        }
+      end
+    end
+    # Parse new submission comments notifications
+    new_submission_comments = []
+    submission_comments_elem = html.at_css("fieldset#messages-comments-submission")
+    if submission_comments_elem
+      submission_comments_elem.css("li:not(.section-controls)").each do |elem|
+        if elem.at_css("input")['checked'] == "checked"
+          if include_deleted
+            new_submission_comments << {
+                comment_id: "",
+                name: "Comment or the submission it was left on has been deleted",
+                profile: "",
+                profile_name: "",
+                is_reply: false,
+                your_submission: false,
+                submission_id: "",
+                title: "Comment or the submission it was left on has been deleted",
+                posted: "",
+                posted_at: ""
+            }
+          end
+          next
+        end
+        elem_links = elem.css("a")
+        date = pick_date(elem.at_css('.popup_date'))
+        new_submission_comments << {
+            comment_id: elem.at_css("input")['value'],
+            name: elem_links[0].content,
+            profile: fa_url(elem_links[0]['href']),
+            profile_name: last_path(elem_links[0]['href']),
+            is_reply: elem.to_s.include?("<em>your</em> comment on"),
+            your_submission: elem.css('em').last.content == "your",
+            submission_id: elem_links[1]['href'].split("/")[-2],
+            title: elem_links[1].content,
+            posted: date,
+            posted_at: to_iso8601(date)
+        }
+      end
+    end
+    # Parse new journal comments notifications
+    new_journal_comments = []
+    journal_comments_elem = html.at_css("fieldset#messages-comments-journals")
```

Comment:
This should be `:fieldset#messages-comments-journal`

Reply:
Oh yeah, fixed in 7d51402f6c396936c147bf5d621fd0428af580b7